pages/blog/_slug.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="max-w-4xl mx-auto">
+    <nui-article class="mt-12 mx-6">
```

Comment:
Is there any value of having `nui-article` component? Why not just native article tag or move complete article block to ArticleBlock or smth?

Reply:
You're right, I will move entire thing to  a main component